### PR TITLE
Note RHEL7, add missing new lines for formatting, spaces for style

### PR DESCRIPTION
**Pull request by edwardcallahan**

### CODE REVIEW

File: src/main/play-doc/gettingStarted/Install.md [hunk 1]
```diff
@@ -25,13 +25,15 @@ The requirements of a ConductR host are:
 * Python 3.4
 * Debian or Rpm package of ConductR
 
+RHEL/CentOS 7 support will be forthcoming.
```
Comment (huntc) [inline src/main/play-doc/gettingStarted/Install.md]: May be we should elaborate on that. What do you think?

Comment (edwardcallahan) [inline src/main/play-doc/gettingStarted/Install.md]: Sure. What further can we definitely state at this time?

Comment (huntc) [inline src/main/play-doc/gettingStarted/Install.md]: Perhaps quote the native packager in terms of Ubuntu only having partial support presently: https://github.com/sbt/sbt-native-packager#experimental-systemd-bootsystem-support

File: src/main/play-doc/gettingStarted/Install.md [hunk 2]
```diff
@@ -25,13 +25,15 @@ The requirements of a ConductR host are:
 * Python 3.4
 * Debian or Rpm package of ConductR
 
+RHEL/CentOS 7 support is waiting for stable [sbt-native-packer systemd support](https://github.com/sbt/sbt-native-packager#experimental-systemd-bootsystem-support).
```
Comment (edwardcallahan) [inline src/main/play-doc/gettingStarted/Install.md:28]: Does this convey enough without getting into details here in the requirements?

Comment (huntc) [inline src/main/play-doc/gettingStarted/Install.md:28]: Yeah. Thanks.

**Pull request merged**